tests/test_app_config.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+from django.test import TestCase
+
+
+class TestConfig(TestCase):
+    def test_config(self):
+        self.assertIsNotNone(settings.SECRET_KEY)
+        self.assertNotEqual(settings.SECRET_KEY, "")
```

Comment:
I don't quite understand this test, is this assuming you have SECRET_KEY as an environment variable? The tests should not depend on environment variables. Will this fail if I don't set up in the GitHub Actions?

Reply:
Umm.. I don't know the other way, can you please elaborate? I guess setting up unit tests for the `SECRET_KEY` variable was the thing to do, is there any other way to do this?

Reply:
> I don't quite understand this test, is this assuming you have SECRET_KEY as an environment variable? The tests should not depend on environment variables. Will this fail if I don't set up in the GitHub Actions?


Yes, it will fail in GitHub actions if `SECRET_KEY` is not exported as an environment variable.
I also don't recommend this test because there is no need of testing this feature @isabelcosta but the issue description consists of writing a test.

Reply:
On mentorship backend we have a test for this, wherein setup of the test we add the variable. but i honestly do not know if this is a good practice or not, for testing environment variables: https://github.com/anitab-org/mentorship-backend/blob/develop/tests/test_app_config.py

I think you could add a setup the secret key on the `os.environ` and another test where you don't do that, and you make sure that the value error gets raised 🤔 

If you think there is no way around this, feel free to remove the test, I am actually not sure here

Reply:
Okay @isabelcosta ! I'll try that.

Reply:
@sahil9001 Are you facing any problems with this?

Reply:
Hey! I'm sorry I got busy with exams, will do it by today!

Reply:
@sahil9001 Is there any problem in implementing @isabelcosta Idea.
I think we need a test for settings.py and if we will add new environment variables in the future then this test can be extended to use that.

Reply:
@isabelcosta said to remove it, so I did, should I write a test case?